### PR TITLE
F22

### PR DESCRIPTION
**Pull request by adimania**
* replaced yum with dnf
* forcing the use of fedora 22 for f22 branch.
* ran the build test from https://gist.github.com/adimania/6965be099904433f626f. Following failed due to package dependencies:
  * firefox
  * container-best-practices
  * haskell

### CODE REVIEW

File: postgresql/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 MAINTAINER Pavel Raiskup <praiskup@redhat.com>
 
 ENV container="docker"
 
-RUN yum -y --setopt=tsflags=nodocs install postgresql-server \
-    && yum -y --setopt=tsflags=nodocs reinstall glibc-common \
-    && yum -y --setopt=tsflags=nodocs clean all --enablerepo='*'
+RUN dnf -y --setopt=tsflags=nodocs install postgresql-server \
+    && dnf -y --setopt=tsflags=nodocs reinstall glibc-common \
+    && dnf -y --setopt=tsflags=nodocs clean all --enablerepo='*'
```
Comment (praiskup) [inline postgresql/Dockerfile]: Its a nit, but pull/125 also fixes this issue.

Comment (adimania) [inline postgresql/Dockerfile]: It does but it is specific to postgres. This pull request is an effort to move the entire branch to dnf and to use fedora 22.

**Comment by dustymabe** [comment]
@adimania as for 4dfe629 I'd like to keep it around in some form. It does represent the most recent version of that package but I agree that it might be misleading to have it on a specific version branch (like F22). Is there a way to keep it around and also not be misleading about the fact that it uses F20?

**Comment by adimania** [comment]
@dustymabe probably we can still keep it in master and not in release specific branches. What do you think?

**Comment by dustymabe** [comment]
@adimania sounds good to me. Eventually I may try to revive the packages and bring them back in Fedora. For now running them in a docker container worked well enough.

**Comment by scollier** [comment]
@adimania i have kicked off a test based on your script.  maybe it'll be finished by tomorrow.

**Comment by scollier** [comment]
LGTM.

**Pull request merged**